src/Components/Locations.js

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect, useState } from "react";
+import LocationList from "./LocationList";
+import "./Locations.css"
+
+export default function Locations() {
+  const [showButton, setShowButton] = useState(false);
+  const [locations, setLocations] = useState([]);
+
+  const handleClick = () => {
+    setShowButton(!showButton);
+  };
+
```

Comment:
1. make a new array from our state [...locations]
2. set up sort functionality - google `js.sort`  => return 1, 0 , -1
3. we want to pass a 'key' to our function
sortLocations(key) a[key] > b[key]
4. set our state again with the new sorted value